contrib/exporters/secadvisor/mod/account_file.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+func makeTimestamp() int64 {
+	return time.Now().UnixNano() / int64(time.Millisecond)
+}
+
+type accountFile struct {
+	filename string
+}
+
+type accountRecord struct {
+	StartupTime  int64 `json:"startup-time"`
+	StartupBytes int64 `json:"startup-bytes"`
+	UpdateTime   int64 `json:"update-time"`
+	UpdateBytes  int64 `json:"update-bytes"`
```

Comment:
Change key to be called `accommodated-bytes` .... can add additional field with 'current-bytes' that has just the current update number of bytes

Reply:
I suggest the following:

- aggregate-start
- aggregate-end
- aggregate-bytes
- current-start
- current-end
- current-bytes